### PR TITLE
Improvement/add package upload support

### PR DESCRIPTION
**Pull request by bspindler**

### CODE REVIEW

File: ts/commands/PackageCommands.ts [hunk 1]
```diff
@@ -42,14 +42,21 @@ class PackageCommands {
       });
 
     (caporal as any)
-      .command('package install', 'Install a package from a Zip URL')
+      .command('package install', 'Install a package from a Zip URL or File')
       .option('--username', 'Metricly Username')
       .option('--password', 'Metricly Password')
       .option('--profile', 'Metricly profile', /.*/, 'default')
-      .argument('<url>', 'Package Download URL')
+      .argument('[url]', 'Package Download URL')
```
Comment (TheConnMan) [inline ts/commands/PackageCommands.ts]: How do you access the second argument without specifying an initial URL argument?

Comment (bspindler) [reply ts/commands/PackageCommands.ts]: right, that's not going to work

Comment (TheConnMan) [reply ts/commands/PackageCommands.ts]: You could take one arg and use https://www.npmjs.com/package/is-url to tell if it's URL or not.

Comment (bspindler) [reply ts/commands/PackageCommands.ts]: I like the idea of letting it detect whether it's a url and fallback to file.  Last night I changed the args to options.  LMK, I think I could be convinced to let it detect vs needing the options.

**Comment by TheConnMan** [comment]
I wanted to see if this package worked and it did, so I guess my answer is yes, it would be cool to automatically decide if it's a URL or file.

**Comment by bspindler** [comment]
awesome!

**Pull request merged**